lib/child_process.js

```diff
@@ -111,28 +111,27 @@ const MAX_BUFFER = 1024 * 1024;
  *   }} [options]
  * @returns {ChildProcess}
  */
-function fork(modulePath /* , args, options */) {
+function fork(modulePath, args = [], options = {}) {
```

Comment:
We could avoid creating a throw away object here.
```suggestion
function fork(modulePath, args = [], options = undefined) {
```

Reply:
> We could avoid creating a throw away object here.

I guess we can just leave off the default value entirely in that case?

```suggestion
function fork(modulePath, args = [], options) {
```